src/libs/ReportUtils.ts

```diff
@@ -1536,20 +1541,18 @@ function requiresAttentionFromCurrentUser(option: OnyxEntry<Report> | OptionData
         return false;
     }
 
-    if (isArchivedRoom(option)) {
-        return false;
-    }
-
-    if (isArchivedRoom(getReport(option.parentReportID))) {
+    if (isArchivedRoom(option) || isArchivedRoom(getReport(option.parentReportID))) {
         return false;
     }
 
-    if (Boolean('isUnreadWithMention' in option && option.isUnreadWithMention) || isUnreadWithMention(option)) {
-        return true;
-    }
+    if (isOption(option)) {
+        if (isUnreadWithMention(option)) {
```

Comment:
`isUnreadWithMention` can have both report and option as the first param. So, is there any reason for restricting it?

Reply:
Agree here, thanks